src/foam/nanos/crunch/SendNotificationOnTopLevelCapabilityStatusUpdate.js

```diff
@@ -40,7 +40,7 @@ foam.CLASS({
 
           // visible checks if the capability is a top-level capability
           // and availabilitypredicate checks if the user has access to the capability
-          if ( cap == null || ! ( cap.getVisible() && cap.getAvailabilityPredicate().f(x) ) ) return;
+          if ( cap == null || ! ( cap.getAvailabilityPredicate().f(x) ) ) return;
```

Comment:
If this condition controls the notification for 'top level capabilities being updated' then I think we have some flawed logic here.

This condition could be checking if the cap has any other dependants (Which might not apply correctly in all scenarios)
OR
A property on capability defines 'top level'. I'm assuming that was the mixed intent with visible.

Reply:
Yeah good thing we do PR reviews. I forgot to update this properly. As long as this rules has a session context it should be easy to fix.